src/client/containers/Channel/Channel.js

```diff
@@ -60,9 +60,9 @@ export default function Channel() {
     );
   }
 
-  const hasTitle = channel?.title !== null;
+  const hasTitle = channel?.title !== 'null' && channel?.title !== null;
```

Comment:
Uhh this looks a little weird. But fine

Reply:
Because in the database we have the channel title string 'null'. I agree this looks a little weird, but ...